securebanking-openbanking-uk-rs-simulator-sample/pom.xml

```diff
@@ -124,6 +104,39 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <configuration>
```

Comment:
This is the equivalent of the Dockerfile in `forgerock-openbanking-uk-aspsp-rs-mock-store-sample`.